src/routes/user/+layout.svelte

```diff
@@ -4,6 +4,7 @@
 	import * as _ from "lodash-es";
 	import { blur } from "svelte/transition";
 	import { Timer as EasyTimer } from "easytimer.js";
+	import { onMount } from "svelte";
```

Comment:
No onmount

Reply:
> No onmount

Sure? lmk the reason?
I'm pretty sure they will introduce some other rune maybe to replace this (on svelte 7 maybe).
for now this LGTM

Reply:
Its deprecated